### PR TITLE
Simplify angle normalization

### PR DESCRIPTION
**Pull request by UniQP**

### CODE REVIEW

**Comment by samueltardieu** [comment]
What problem does it solve exactly? From what I can tell, no new test has been introduced.

**Comment by UniQP** [comment]
What commit are you talking about?

**Comment by UniQP** [comment]
I added an additional test that would fail with `assertThat`. I hope that's what you asked for.

File: main/src/cgeo/geocaching/utils/AngleUtils.java [hunk 1]
```diff
@@ -35,7 +35,8 @@ public static float difference(final float from, final float to) {
      * @return the same angle in the [0, 360[ range
      */
     public static float normalize(final float angle) {
-        return (angle >= 0 ? angle : (360 - ((-angle) % 360))) % 360;
+        final float mod = angle % 360;
+        return mod >= 0 ? mod : (mod + 360) % 360;
```
Comment (samueltardieu) [inline main/src/cgeo/geocaching/utils/AngleUtils.java]: Is the `% 360` needed here?

Comment (UniQP) [inline main/src/cgeo/geocaching/utils/AngleUtils.java]: Yes, because -Float.MIN_VALUE + 360 becomes 360, which is not within the desired interval.

Comment (samueltardieu) [inline main/src/cgeo/geocaching/utils/AngleUtils.java]: Indeed.

**Pull request merged**